### PR TITLE
Update Curb Examples

### PR DESCRIPTION
**Pull request by mplsmitch**
Minor corrections to JSON and formatting

---
name: Mitch Vars

title: Minor corrections to JSON and formatting

---

# CDS Pull Request

## Explain pull request

Minor corrections to JSON examples flagged in #111. Removes an extraneous array[ ] and adds missing rate objects {}.
The JSON examples on this page are not valid. This PR fixes them.
## Is this a breaking change

* No, not breaking

### CODE REVIEW

File: curbs/examples.md [hunk 1]
```diff
@@ -26,31 +26,46 @@ A [Query Curb Zones](/curbs#query-curb-zones) example of `/curbs/zones` with min
...
-      ] 
-    }
-  ]
+  "data": {
+    "zones": [
+      {
+        "curb_zone_id": "7d8a5885-e949-4ac9-afb7-fa4d43b68530",
+        "geometry": {
+          "type": "Polygon",
+          "coordinates": [
```
Comment (schnuerle) [inline curbs/examples.md:35]: Can you format this down to this just to be more compact on the page? 

            "coordinates": [[
              [-73.982105, 40.767932],
              [-73.973694, 40.764551],
              [-73.949318, 40.796918],
              [-73.958416, 40.800686],
              [-73.982105, 40.767932]
            ]]

**Comment by schnuerle** [comment]
Can you also please update the Events examples? https://github.com/openmobilityfoundation/curb-data-specification/blob/main/events/examples.md

File: curbs/examples.md (hunk 1 above)
Comment (mplsmitch) [reply curbs/examples.md]: Done in [a2a694a](https://github.com/openmobilityfoundation/curb-data-specification/pull/132/commits/a2a694ab9210a7dae1fc325bb7739a975c92e38c)

File: curbs/examples.md [hunk 2]
```diff
@@ -158,33 +163,38 @@ A [Query Curb Policies](/curbs#query-curb-policies) example of `/curbs/policies`
         {
```
Comment (schnuerle) [inline curbs/examples.md:163]: In this third example, the `"data": [` array needs to be removed too.

Comment (mplsmitch) [reply curbs/examples.md]: fixed in [942d9b0](https://github.com/openmobilityfoundation/curb-data-specification/pull/132/commits/942d9b05456e09bac16e00f20014e9b3bf4d9158)

**Pull request merged**